{
  "uid": "3d25c7768928a23e",
  "name": "test_sub_unusual[sub_unusual_data1]",
  "fullName": "test_calc.TestSub#test_sub_unusual",
  "historyId": "a064d5fdbb9562c0b477cd715b2eed99",
  "time": {
    "start": 1597675959392,
    "stop": 1597675959392,
    "duration": 0
  },
  "status": "skipped",
  "statusMessage": "TypeError: unsupported operand type(s) for -: 'str' and 'float'",
  "statusTrace": "self = <Hogwarts_pytest_zuoye.testcode.test_calc.TestSub object at 0x000001FEF971C8D0>, calc_init = <sourcecode.calc_code.Calculator object at 0x000001FEF96C42B0>, sub_unusual_data = ['r', 0.2, 'rt']\n\n    @pytest.mark.xfail(raises=TypeError)  # 预估返回类型错误\n    def test_sub_unusual(self, calc_init, sub_unusual_data):\n>       result = calc_init.sub(sub_unusual_data[0], sub_unusual_data[1])\n\ntest_calc.py:72: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <sourcecode.calc_code.Calculator object at 0x000001FEF96C42B0>, a = 'r', b = 0.2\n\n    def sub(self, a, b):\n>       return a - b\nE       TypeError: unsupported operand type(s) for -: 'str' and 'float'\n\n..\\sourcecode\\calc_code.py:8: TypeError",
  "flaky": false,
  "newFailed": false,
  "beforeStages": [
    {
      "name": "calc_init",
      "time": {
        "start": 1597675959371,
        "stop": 1597675959371,
        "duration": 0
      },
      "status": "passed",
      "steps": [],
      "attachments": [],
      "parameters": [],
      "shouldDisplayMessage": false,
      "attachmentsCount": 0,
      "hasContent": false,
      "stepsCount": 0
    },
    {
      "name": "sub_unusual_data",
      "time": {
        "start": 1597675959392,
        "stop": 1597675959392,
        "duration": 0
      },
      "status": "passed",
      "steps": [],
      "attachments": [],
      "parameters": [],
      "shouldDisplayMessage": false,
      "attachmentsCount": 0,
      "hasContent": false,
      "stepsCount": 0
    }
  ],
  "afterStages": [
    {
      "name": "calc_init::0",
      "time": {
        "start": 1597675959405,
        "stop": 1597675959405,
        "duration": 0
      },
      "status": "passed",
      "steps": [],
      "attachments": [],
      "parameters": [],
      "shouldDisplayMessage": false,
      "attachmentsCount": 0,
      "hasContent": false,
      "stepsCount": 0
    }
  ],
  "labels": [
    {
      "name": "tag",
      "value": "run(order=2)"
    },
    {
      "name": "tag",
      "value": "@pytest.mark.xfail(raises=<class 'TypeError'>)"
    },
    {
      "name": "tag",
      "value": "sub"
    },
    {
      "name": "suite",
      "value": "test_calc"
    },
    {
      "name": "subSuite",
      "value": "TestSub"
    },
    {
      "name": "host",
      "value": "TJ-YF-262JLJ"
    },
    {
      "name": "thread",
      "value": "4968-MainThread"
    },
    {
      "name": "framework",
      "value": "pytest"
    },
    {
      "name": "language",
      "value": "cpython3"
    },
    {
      "name": "package",
      "value": "test_calc"
    },
    {
      "name": "resultFormat",
      "value": "allure2"
    }
  ],
  "parameters": [
    {
      "name": "sub_unusual_data",
      "value": "['r', 0.2, 'rt']"
    }
  ],
  "links": [],
  "hidden": false,
  "retry": false,
  "extra": {
    "severity": "normal",
    "retries": [],
    "categories": [],
    "tags": [
      "run(order=2)",
      "sub",
      "@pytest.mark.xfail(raises=<class 'TypeError'>)"
    ]
  },
  "source": "3d25c7768928a23e.json",
  "parameterValues": [
    "['r', 0.2, 'rt']"
  ]
}